cmd/routed-eni-cni-plugin/cni.go

```diff
@@ -241,12 +249,15 @@ func add(args *skel.CmdArgs, cniTypes typeswrapper.CNITYPES, grpcClient grpcwrap
 
 	hostInterface := &current.Interface{Name: hostVethName}
 	containerInterface := &current.Interface{Name: args.IfName, Sandbox: args.Netns}
+	vlanInterface := &current.Interface{Name: vlanInterfaceName, Mac: fmt.Sprint(r.PodVlanId)}
+	log.Infof("Using vlanInterface: %v", vlanInterface)
 
```

Comment:
For the final commit, let's explore if the actual vlan interface name can be used  rather than place holder name (const string). This may or may not affect CNI chaining use cases if we have duplicate intf name across SGPP containers.

Reply:
Yes, I have made a note of TODO changes